multipaz-zkp/src/commonTest/kotlin/org/multipaz/zkp/MdocTestDataProvider.kt

```diff
@@ -0,0 +1,273 @@
+            0x8a, 0x1a, 0xc9, 0x28, 0xa6, 0x28, 0x2c, 0xfa, 0x6d, 0x64, 0x65, 0x76, 0x69, 0x63, 0x65,
+            0x4b, 0x65, 0x79, 0x49, 0x6e, 0x66, 0x6f, 0xa1, 0x69, 0x64, 0x65, 0x76, 0x69, 0x63, 0x65,
+            0x4b, 0x65, 0x79, 0xa4, 0x01, 0x02, 0x20, 0x01, 0x21, 0x58, 0x20, 0xc2, 0x41, 0xc4, 0x0e,
+            0x81, 0x64, 0xf1, 0xb3, 0xa7, 0xa3, 0x90, 0xf7, 0x16, 0xc0, 0x09, 0xcf, 0x54, 0x3f, 0x5e,
+            0xe7, 0x36, 0xd9, 0xc6, 0x65, 0xe6, 0x01, 0x24, 0xa4, 0x98, 0xdb, 0xbb, 0x4d, 0x22, 0x58,
+            0x20, 0x7d, 0xd6, 0x8b, 0x7b, 0x54, 0x05, 0x16, 0x1a, 0x03, 0x3a, 0xc5, 0xf1, 0x5f, 0xe2,
+            0x8c, 0x4a, 0xdd, 0xef, 0x3f, 0xfd, 0x2e, 0xd4, 0x41, 0xb7, 0x10, 0x15, 0x8f, 0xca, 0xc5,
+            0x80, 0xe0, 0x62, 0x6c, 0x76, 0x61, 0x6c, 0x69, 0x64, 0x69, 0x74, 0x79, 0x49, 0x6e, 0x66,
+            0x6f, 0xa3, 0x66, 0x73, 0x69, 0x67, 0x6e, 0x65, 0x64, 0xc0, 0x74, 0x32, 0x30, 0x32, 0x34,
+            0x2d, 0x31, 0x31, 0x2d, 0x30, 0x33, 0x54, 0x32, 0x31, 0x3a, 0x31, 0x39, 0x3a, 0x31, 0x39,
+            0x5a, 0x69, 0x76, 0x61, 0x6c, 0x69, 0x64, 0x46, 0x72, 0x6f, 0x6d, 0xc0, 0x74, 0x32, 0x30,
+            0x32, 0x34, 0x2d, 0x31, 0x31, 0x2d, 0x30, 0x33, 0x54, 0x32, 0x31, 0x3a, 0x31, 0x39, 0x3a,
+            0x31, 0x39, 0x5a, 0x6a, 0x76, 0x61, 0x6c, 0x69, 0x64, 0x55, 0x6e, 0x74, 0x69, 0x6c, 0xc0,
+            0x74, 0x32, 0x30, 0x32, 0x34, 0x2d, 0x31, 0x32, 0x2d, 0x30, 0x33, 0x54, 0x32, 0x31, 0x3a,
+            0x31, 0x39, 0x3a, 0x31, 0x39, 0x5a, 0x58, 0x40, 0xd3, 0x80, 0x2c, 0xdb, 0xb2, 0x24, 0xa3,
+            0xfe, 0x40, 0x84, 0xe8, 0x47, 0x8a, 0x83, 0x50, 0xbf, 0x8d, 0x55, 0x70, 0x90, 0xcc, 0xfb,
+            0x36, 0x2b, 0x88, 0xb9, 0xf5, 0xcb, 0xb8, 0xea, 0xfe, 0xaa, 0x16, 0xa7, 0x50, 0xac, 0x72,
+            0x69, 0x75, 0x62, 0x32, 0xea, 0x09, 0xc6, 0xbb, 0xe9, 0x4b, 0xee, 0x00, 0x47, 0x4f, 0x8d,
+            0xc2, 0x80, 0x0e, 0x0b, 0xcd, 0x69, 0xdc, 0xc7, 0x66, 0xcf, 0x07, 0xf5, 0x6c, 0x64, 0x65,
+            0x76, 0x69, 0x63, 0x65, 0x53, 0x69, 0x67, 0x6e, 0x65, 0x64, 0xa2, 0x6a, 0x6e, 0x61, 0x6d,
+            0x65, 0x53, 0x70, 0x61, 0x63, 0x65, 0x73, 0xd8, 0x18, 0x41, 0xa0, 0x6a, 0x64, 0x65, 0x76,
+            0x69, 0x63, 0x65, 0x41, 0x75, 0x74, 0x68, 0xa1, 0x6f, 0x64, 0x65, 0x76, 0x69, 0x63, 0x65,
+            0x53, 0x69, 0x67, 0x6e, 0x61, 0x74, 0x75, 0x72, 0x65, 0x84, 0x43, 0xa1, 0x01, 0x26, 0xa0,
+            0xf6, 0x58, 0x40, 0x4d, 0x08, 0x42, 0xad, 0x45, 0x34, 0x88, 0xb7, 0x1a, 0x1d, 0x59, 0x3e,
+            0xae, 0x47, 0x94, 0xfe, 0x1a, 0x40, 0xe7, 0x50, 0xa2, 0x10, 0xcc, 0x51, 0xb9, 0xae, 0xe7,
+            0xaf, 0xcf, 0x9c, 0xae, 0xd6, 0x2f, 0x14, 0x08, 0xea, 0xfa, 0x2a, 0x23, 0x61, 0x49, 0xa2,
+            0x93, 0x5b, 0x0c, 0x2f, 0xb6, 0x4c, 0x29, 0x02, 0xa8, 0x51, 0x6b, 0x32, 0xef, 0xce, 0x08,
+            0x2b, 0x7d, 0x09, 0x94, 0x61, 0x9a, 0x82, 0x66, 0x73, 0x74, 0x61, 0x74, 0x75, 0x73, 0x00
+        ))
+    }
+}
```

Comment:
Is this actually used anywhere? If it's for a future commit, suggest to include it in that future commit..

Reply:
This is used in the unit tests for the native bridge